src/Metadata/ApiProperty.php

```diff
@@ -205,6 +212,7 @@ public function __construct(
         private ?string $uriTemplate = null,
         private ?string $property = null,
         private ?string $policy = null,
+        private ?array $serialize = null,
```

Comment:
```suggestion
        private ?(array|Context|Groups|Ignore|SerializedName|SerializedPath|MaxDepth) $serialize = null,
```

And cast it in the constructor.

Nit: for this parameter and for `$types`, it would be stricter to use a standard parameter and create a separate property that is of type `?array` (because we cast them to array in the constructor itself).